rpc/cc/arduino/cli/commands/v1/core.proto

```diff
@@ -33,6 +33,9 @@ message PlatformInstallRequest {
   // Set to true to not run (eventual) post install scripts for trusted
   // platforms
   bool skip_post_install = 5;
+  // Set to true will not install the platform if another version is already
+  // installed.
```

Comment:
```suggestion
  // Set to true to skip installation if a different version of the platform
  // is already installed.
```
Align documentation comment with that of `cc.arduino.cli.commands.v1.ArduinoCoreService.LibraryInstall`